internal/auth/options.go

```diff
@@ -100,21 +85,11 @@ type Options struct {
 	GroupsCacheRefreshTTL time.Duration `mapstructure:"groups_cache_refresh_ttl"`
 	SessionLifetimeTTL    time.Duration `mapstructure:"session_lifetime_ttl"`
 
-	PassHostHeader     bool `mapstructure:"pass_host_header"`
-	SkipProviderButton bool `mapstructure:"skip_provider_button"`
-	PassUserHeaders    bool `mapstructure:"pass_user_headers"`
-	SetXAuthRequest    bool `mapstructure:"set_xauthrequest"`
-
 	// These options allow for other providers besides Google, with potential overrides.
 	Provider         string `mapstructure:"provider"`
 	ProviderSlug     string `mapstructure:"provider_slug"`
 	ProviderServerID string `mapstructure:"provider_server_id"`
 
-	SignInURL      string `mapstructure:"signin_url"`
-	RedeemURL      string `mapstructure:"redeem_url"`
-	RevokeURL      string `mapstructure:"revoke_url"`
-	ProfileURL     string `mapstructure:"profile_url"`
-	ValidateURL    string `mapstructure:"validate_url"`
```

Comment:
With these removed, do we still want to provide the functionality for custom URL's to be passed in to providers? e.g: https://github.com/buzzfeed/sso/blob/master/internal/auth/providers/google.go#L45

Reply:
That is my intent, yes -- I don't think this configuration is very useful if we also have to provide it with a provider implementation.

Reply:
Great, agreed!